textBasedAdventure.js

```diff
@@ -1,5 +1,457 @@
-const readline = require('readline-sync')
+  
+const readline = require('readline-sync')           // readline
+var readlineSync = require('readline-sync');        // readlineSync
 
+// Input name
 let nameInput = readline.question("Enter your name: ")
+console.log(`Hello ${nameInput}! 
+██╗    ███████████╗     ██████╗██████╗███╗   ██████████╗    ████████╗██████╗     ███╗   █████╗   ██╗     ██████╗ █████╗███╗   ██████████╗
+██║    ████╔════██║    ██╔════██╔═══██████╗ ██████╔════╝    ╚══██╔══██╔═══██╗    ████╗ ████╚██╗ ██╔╝    ██╔════╝██╔══██████╗ ██████╔════╝
+██║ █╗ ███████╗ ██║    ██║    ██║   ████╔████╔███████╗         ██║  ██║   ██║    ██╔████╔██║╚████╔╝     ██║  ████████████╔████╔███████╗  
+██║███╗████╔══╝ ██║    ██║    ██║   ████║╚██╔╝████╔══╝         ██║  ██║   ██║    ██║╚██╔╝██║ ╚██╔╝      ██║   ████╔══████║╚██╔╝████╔══╝  
+╚███╔███╔██████████████╚██████╚██████╔██║ ╚═╝ █████████╗       ██║  ╚██████╔╝    ██║ ╚═╝ ██║  ██║       ╚██████╔██║  ████║ ╚═╝ █████████╗
+ ╚══╝╚══╝╚══════╚══════╝╚═════╝╚═════╝╚═╝     ╚═╚══════╝       ╚═╝   ╚═════╝     ╚═╝     ╚═╝  ╚═╝        ╚═════╝╚═╝  ╚═╚═╝     ╚═╚══════╝
+                                                                                                                                         
+`)
 
```

Comment:
Maybe put all the assets/text at the bottom so when you scan the file you don’t get distracted.